rtt/base/ChannelElement.hpp

```diff
@@ -63,6 +63,8 @@ namespace RTT { namespace base {
         typedef typename boost::call_traits<T>::param_type param_t;
         typedef typename boost::call_traits<T>::reference reference_t;
 
+        virtual ~ChannelElement() { }
```

Comment:
The base class already has a virtual destructor and the implicitly defined destructor is therefore also `virtual`. You could add an explicit definition, but it does not change anything related to potential memory leaks and might break the ABI, so I would rather keep those class declarations as-is.

Reply:
Reverting the (redundant) added explicit destructors, nothing of value is added by those.